app/components/common/GridWrapper.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import styled from 'styled-components';
+
+interface GridWrapperProps {
+    /**
+     * text content
+     */
+    children?: React.ReactNode;
+}
+
+const StyledDiv = styled.div<GridWrapperProps>`
+    display: grid;
+    grid-template-columns: repeat(auto-fill, minmax(300px, 3fr));
+    grid-gap: 20px;
+    margin-bottom: 1rem;
+`;
+
+const GridWrapper = ({ children, ...rest }: GridWrapperProps) => <StyledDiv {...rest}>{children}</StyledDiv>;
+
+export default GridWrapper;
```

Comment:
WDYT about the following simplification

```suggestion
import styled from 'styled-components';

const GridWrapper = styled.div`
    display: grid;
    grid-template-columns: repeat(auto-fill, minmax(300px, 3fr));
    grid-gap: 20px;
    margin-bottom: 1rem;
`;

export default GridWrapper;
```

We are trying to create additional interfaces/functions for passing down properties natively supported by `styled-components
Because of that I was wondering about using just it :)

Let me know WDYT 😉

Reply:
https://github.com/cloudify-cosmo/cloudify-stage/pull/2465/commits/2a026e525cd2c8a6dcb30844e4f61c48f5a85e32 implemented ✔️